### PR TITLE
Arithmetic support for numerical input fields

### PR DESCRIPTION
**Pull request by tandej**
Let's try this again...

Simply upstreaming the changes from c9814a5, but git rebase crapped itself the first time :(

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,6 +32,7 @@
     "@tauri-apps/api": "^1.4.0",
     "ajv": "^8.12.0",
     "d3": "^7.8.5",
+    "mathjs": "^11.9.1",
```
Comment (jlbabilino) [inline package.json:35]: This should be version `12.3.0`

File: src-tauri/libdummy.so* [hunk 2]
```diff
@@ -0,0 +1 @@
+dummy file (see "scripts/build_utils.cjs")
```
Comment (jlbabilino) [inline src-tauri/libdummy.so*:1]: Don't commit this

File: src/components/input/Input.tsx [hunk 3]
```diff
@@ -143,7 +144,7 @@ class Input extends Component<Props, State> {
             this.focusedMode();
           }}
           onBlur={(e) => {
-            let newNumber = parseFloat(this.state.editedValue);
+            let newNumber = parseFloat(parse(this.state.editedValue).evaluate().toString());
```
Comment (jlbabilino) [inline src/components/input/Input.tsx:147]: Make sure to run `npm run fixFormat`

Comment (shueja) [inline src/components/input/Input.tsx]: If this is parsed as numeric by mathjs we don’t need an extra toString and parseFloat here

**Review by shueja — CHANGES_REQUESTED** [review]

**Comment by jlbabilino** [comment]
Is this a new issue?

https://github.com/SleipnirGroup/Choreo/assets/28580376/fedcb5fa-743e-436b-a3d2-96765ee8aa0b

**Comment by shueja** [comment]
Superseded by #562

**Pull request closed without merging**